javascript/modules/client/test/integration/client.test.ts

```diff
@@ -450,10 +569,79 @@ describe("Client", () => {
     });
   });
   describe("Action decoders", () => {
-    it("Should decode an encoded raw withdraw action of an erc20 token", async () => {
+    it("Should decode an encoded grant action", () => {
+      const context = new Context(contextParamsLocalChain);
+      const client = new Client(context);
+
+      const grantParams: IPermissionParams = {
+        who: "0x1234567890123456789012345678901234567890",
+        where: "0x1234567890123456789012345678901234567890",
+        permission: Permissions.UPGRADE_PERMISSION,
+      };
```

Comment:
Could we have a test vector, with at least 2 different parameter sets being checked?

Reply:
We aren't doing this yet